documentation/src/main/markdown/gettingstarted/terms/tools.md

```diff
@@ -1,12 +1,12 @@
-# [solution_name] tools
+# Tools
 
-[solution_name] tools are run to enable the scanning of your code.
+[solution_name] tools are components that enable the scanning of your source code. [solution_name] uses several underlying tools to perform scanning including the Detector (for inspecting package manager dependencies), Signature Scanner (for inspecting the file system), and others. Optional properties can be specified to explicitly enable or disable these underlying tools, but by default [solution_name] will run both the Detector and Signature Scanner on the code being analyzed.
```

Comment:
We should replace 'and others' with the definite list of tools that Detect uses.

Reply:
Can do. (I'll add what I know, and then ask if you can please confirm.)

Reply:
@gopannabd Done.